lib/types/locale.ts

```diff
@@ -52,6 +52,7 @@ export interface Locale {
     sunday?: string;
     feast?: string;
     memorial?: string;
+    optional_memorial?: string;
     weekday?: string;
   };
```

Comment:
No, because an XX-XX locale may extend an XX locale. So, at this level, it should remain optional (the script added here #461 will, however, ensure that all these items are correctly defined and present for a language).

Reply:
You’re right, I have forgotten about the regional locales. :+1: